server/models/User.js

```diff
@@ -0,0 +1,18 @@
+import mongoose from 'mongoose';
+
+const UserSchema = new mongoose.Schema({
```

Comment:
please add timestamps ( createdAt + updatedAt ) into the [schema options](https://mongoosejs.com/docs/guide.html#timestamps)